src/docs/components/Code/TabbedCode.tsx

```diff
@@ -17,100 +21,62 @@ export interface TabbedCodeItem {
 interface TabbedCodeProps {
   className?: string;
   collapsedClassName?: string;
-  expandedClassName?: string;
   tabs: TabbedCodeItem[];
 }
 
-const CodeExpandableFooter = ({
-  expanded,
-  setExpanded,
-}: {
-  expanded: boolean;
-  setExpanded: (value: boolean) => void;
-}) => (
-  <footer
-    className={cn(
-      "absolute pointer-events-none rounded-md h-full w-full inset-0 bg-gradient-to-t from-slate-900 flex justify-center items-end pb-8",
-      expanded ? "pb-2 bg-none" : "pb-8"
-    )}
-  >
-    <button
-      onClick={() => setExpanded(!expanded)}
-      className={cn([
-        expanded
-          ? "text-xs px-2 py-1"
-          : "text-sm px-3 py-1.5 bg-neutral-200 text-neutral-800",
-        "pointer-events-auto rounded-md font-medium transition-all duration-300",
-        /** Hover styles */
-        expanded
-          ? "hover:bg-slate-800 hover:text-white"
-          : "hover:bg-neutral-400",
-        /** Focus styles */
-        "focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-dark-400 focus-visible:ring-offset-2 ring-offset-dark-800",
-        /** Disabled styles */
-        "disabled:pointer-events-none disabled:opacity-50",
-      ])}
-    >
-      {!expanded ? "Expand" : "Collapse"}
-    </button>
-  </footer>
-);
-
 export const TabbedCode = ({
   collapsedClassName,
-  expandedClassName,
   className,
   tabs,
 }: TabbedCodeProps) => {
-  const isExpandable = collapsedClassName || expandedClassName;
+  const isExpandable = typeof collapsedClassName === "string";
+  const Wrapper = isExpandable ? FadingCollapsible : React.Fragment;
```

Comment:
This Wrapper related code is bit Vueish way of writing related logic. Is it okay in React?

Reply:
Yes, this is also a valid pattern in React.